docs/data/data-grid/aggregation/AggregationColDefAggregable.js

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { DataGridPremium } from '@mui/x-data-grid-premium';
+import { useMovieData } from '@mui/x-data-grid-generator';
+
+const COLUMNS = [
+  {
+    field: 'title',
+    headerName: 'Title',
+    width: 200,
+    groupable: false,
+    aggregable: false,
+  },
+  {
+    field: 'gross',
+    headerName: 'Gross',
+    type: 'number',
+    width: 150,
+    groupable: false,
+    valueFormatter: ({ value }) => {
+      if (!value || typeof value !== 'number') {
+        return value;
+      }
+      return `${value.toLocaleString()}$`;
+    },
+  },
+  {
+    field: 'year',
+    headerName: 'Year',
+    type: 'number',
+    aggregable: false,
+  },
+];
+
+export default function AggregationColDefAggregable() {
```

Comment:
```suggestion
export default function AggregationColDefAggregatable() {
```

Reply:
We went for `aggregable` on the pro (after a discussion with Sam)